Commands/MakeFilter.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace Vulcan\Commands;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use Vulcan\Libraries\GeneratorTrait;
+
+/**
+ * Creates a skeleton Filter.
+ *
+ * @package Vulcan\Commands
+ */
+class MakeFilter extends BaseCommand
+{
+	use GeneratorTrait;
+
+	protected $group = 'Vulcan';
+
+	/**
+	 * The Command's name
+	 *
+	 * @var string
+	 */
+	protected $name = 'make:filter';
+
+	/**
+	 * the Command's short description
+	 *
+	 * @var string
+	 */
+	protected $description = 'Creates a skeleton Filter class.';
```

Comment:
Since we have both Controller filters and Parser Filters, we need to specify which one here.